tmk_core/common/chibios/suspend.c

```diff
@@ -43,6 +50,16 @@ void suspend_power_down(void) {
     // TODO: figure out what to power down and how
     // shouldn't power down TPM/FTM if we want a breathing LED
     // also shouldn't power down USB
+#if defined(RGBLIGHT_SLEEP) && defined(RGBLIGHT_ENABLE)
+#    ifdef RGBLIGHT_ANIMATIONS
+    rgblight_timer_disable();
```

Comment:
So this was in the AVR code, but I don't seem to actually need it since rgblight_disable_noeeprom appears to disable the timer itself. Should I remove it?